cmd/lora-gateway-bridge/cmd/configfile.go

```diff
@@ -112,6 +112,12 @@ type="{{ .Backend.Type }}"
   # certificate of the gateway has been signed by this CA certificate.
   ca_cert="{{ .Backend.BasicStation.CACert }}"
 
+  # Verify client vertificate CommonName
+  #
+  # Require that the CommonName on the client certificate matches
+  # the EUI that the gateway is claiming to be.
+  verify_cn={{ .Backend.BasicStation.VerifyCN }}
```

Comment:
See my comment in the conversation. Maybe we should always validate the CN when a client certificate is being used, and not make this a config option. What do you think?

Reply:
I've removed the verify_cn option, as per my other comment, I think it's a good idea.